src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbBasicTest.scala

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.dsde.rawls.graph
```

Comment:
I figured we should have a separate package for graph-specific implementation details / classes. The DAO would still be in dataaccess, but there's lots of other stuff that we will need to have, and where should that go?